resources/chatconfig.ini

```
@@ -6,3 +6,11 @@ max_number_of_best_friends=5
 # Change the value below to what you would like this to be (50 is live accurate)
 # going over 50 will be allowed in some secnarios, but proper handling will require client modding
 max_number_of_friends=50
+
+# Enable or disable the chat web API, disabled by default
+# It will run on the same port the chat server is running on, defined in shardconfig.ini
+enable_chat_web_api=0
+
+# If that chat web api is enabled, it will only listen for connections on this ip address
+# 127.0.0.1 is localhost
+chat_web_api_listen_address=127.0.0.1
```

```suggestion
chat_web_api_listen_address=localhost
```
Raknet should translate local host to this ip, more readable than the ip imo.

---

httplib needs and IP to listen on, localhost does not work